ValetTests/ValetTests.m

```diff
@@ -90,15 +91,20 @@ - (void)setUp;
     self.string = @"bar";
     self.secondaryString = @"bar2";
     self.additionalValets = [NSMutableArray new];
+    self.manualKeychainEntries = [NSMutableArray<NSDictionary *> new];
 }
 
 - (void)tearDown;
 {
-    [super tearDown];
-    
     for (VALValet *const additionalValet in self.additionalValets) {
         [additionalValet removeAllObjects];
     }
+
+    for (NSDictionary *keychainEntry in self.manualKeychainEntries) {
```

Comment:
I'm throwing straw men out there now (but I have been bit by these in the past): `finally` won't prevent against
- Hitting cmd+. in the middle of the test
- Another test polluting the keychain

Normally I don't protect against these kinds of issues in my tests. But since keychain is both persistent and a black box I like to control for as many factors as I can in each test.